rusoto/services/apigateway/src/generated.rs

```diff
@@ -17114,7 +17114,7 @@ where
         request.set_content_type("application/x-amz-json-1.1".to_owned());
 
         let future = self.inner.sign_and_dispatch(request).and_then(|response| {
-            if response.status == StatusCode::Ok {
+            if response.status.is_success() {
```

Comment:
This file looks like it shouldn't have `use hyper::StatusCode;` but there are some parts that *didn't* change, such as `if response.status == StatusCode::Created {` in `create_base_path_mapping`. 😄